engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -397,6 +398,9 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
       } else {
         goto(WorkflowExecutionAbortedState)
       }
+    case Event(EngineStatsActor.JobCountQuery, data) =>
+      sender ! EngineStatsActor.JobCount(data.backendJobExecutionActors.size)
```

Comment:
I view this as a different statistic. I'll take a quick look to see if it's easy to grab that as well. If not I'll leave it for future expansion.